docs/build/zkEVM/zk-node/setup-local-validium-node.md

```diff
@@ -0,0 +1,335 @@
+---
+sidebar_position: 1
+title: Setup Local Validium Node
+sidebar_label: Setup Local Validium Node
+---
+
+import cdk1 from '/docs/build/zkEVM/img/add-cdk-1.png'
+import cdk2 from '/docs/build/zkEVM/img/add-cdk-2.png'
+import cdk3 from '/docs/build/zkEVM/img/add-cdk-3.png'
+import cdk4 from '/docs/build/zkEVM/img/add-cdk-4.png'
+import cdk5 from '/docs/build/zkEVM/img/add-cdk-5.gif'
+import cdk6 from '/docs/build/zkEVM/img/add-cdk-6.gif'
+import cdk7 from '/docs/build/zkEVM/img/add-cdk-7.png'
+import cdk8 from '/docs/build/zkEVM/img/add-cdk-8.png'
+import cdk9 from '/docs/build/zkEVM/img/add-cdk-9.png'
+import cdk10 from '/docs/build/zkEVM/img/add-cdk-10.png'
+import cdk11 from '/docs/build/zkEVM/img/add-cdk-11.png'
+import cdk12 from '/docs/build/zkEVM/img/add-cdk-12.png'
+import cdk13 from '/docs/build/zkEVM/img/add-cdk-13.png'
+import cdk14 from '/docs/build/zkEVM/img/add-cdk-14.png'
+import cdk15 from '/docs/build/zkEVM/img/add-cdk-15.png'
+
+
+## Validium
+
+This quick start guide shows you how to set up a CDK validium on your local machine that sets up and runs the following components:
+
+- zkEVM databases: data node, event, explorer L1 and L2, pool, state, and bridge service
+- zkEVM node components: aggregator, approve service, sequencer and sequence sender, synchronizer
+- L1 network (mock)
+- Prover
+- Explorers L1, L2
+- JSON RPC explorer
+- L2 gas pricer
+- DAC: data availability service, dac setup committee
+- zkEVM bridge service and UI
+
+:::note 
+
+The documentation describes standard deployments. 
+Edit the configuration files to implement your own custom setups.
+
+:::
+
+## Prerequisites
+
+### Hardware Requirements
+
+- A Linux-based OS (e.g., Ubuntu Server 22.04 LTS).
+- At least 16GB RAM with a 4-core CPU.
+- An AMD64 architecture system.
+
+:::caution
+
+Currently the zkProver does not run on ARM-powered Macs. For Windows users, using WSL/WSL2 is not recommended.
+
+Unfortunately, Apple M1 chips are not supported for now - since some optimizations on the zkProver require specific Intel instructions. This means some non-M1 computers won't work regardless of the OS, for example: AMD.
```

Comment:
@mc2eqe should it be fair to say just Apple M-series chips are not supported (it;s not just M1 but M1,2,3) ?

Reply:
Yes it would be. 
My question: is it ok POL as a currency symbol in pictures? It's obviously Polygon's product and software.

Reply:
For now it's fine.
Counter question for long term - Would a config change be simple to run with some provisional token name ("ASTR" or whatever) - if simple, we could perhaps adjust with lower prio